doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/security-management-rest-api-permissions-ref.adoc

```diff
@@ -0,0 +1,370 @@
+[id='security-management-rest-api-permissions-ref_{context}']
+= Permissions
+
+The Security Management REST API supports the following endpoints for managing permissions granted to the groups, roles and users in {CENTRAL}. The Security Management REST API base URL is `http://SERVER:PORT/business-central/rest/`. All requests require HTTP Basic authentication or token-based authentication for the `admin` user role.
```

Comment:
Per IBM SG, always use serial comma, so should be "... to the groups, roles, and users in ..."  I've seen it at least one other place. Suggest a quick look through your section intros to confirm, and bear in mind going forward.